data/brands/amenity/restaurant.json

```diff
@@ -1012,11 +1012,20 @@
       "tags": {
         "amenity": "restaurant",
         "brand": "Din Tai Fung",
+        "brand": "鼎泰豐",
         "brand:wikidata": "Q699142",
         "brand:wikipedia": "en:Din Tai Fung",
         "cuisine": "taiwanese",
-        "name": "Din Tai Fung",
-        "name:zh": "鼎泰丰"
+        "name": "鼎泰豐",
```

Comment:
Din Tai Fung has locations in several countries (listed above), most of which do not use Chinese or Min Nan as their primary languages. As far as I can tell, this entry was originally written for the situation in countries like the United States, where `name` would be in pinyin and `name:zh` would still be tagged (since the chain does have a Chinese name). Since the situation is obviously different in Taiwan, we should add a separate entry for the same brand with `name` set to Chinese, alongside the existing one in pinyin. The two entries can have different `locationSet`s.